modules/update-service-create-service-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+[id="update-service-create-service-web-console_{context}"]
+= Creating an OpenShift Update Service application by using the web console
+
+You can use the {product-title} web console to create an Update Service application by using the OpenShift Update Service Operator.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
```

Comment:
`The Openshift Update Service graph data container image has already been created and pushed to local registry` is also a prerequisite.

Reply:
Fixed